models/twitter_test.go

```diff
@@ -0,0 +1,41 @@
+package models
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestTwitterUserList(t *testing.T) {
+	list := &TwitterUserList{}
+	assert.Equal(t, len(*list), 0, "should not contain any elements before create")
+
+	slice1 := []*TwitterUser{
+		&TwitterUser{
+			Username: "muesli",
+		},
+		&TwitterUser{
+			Username: "smag",
+		},
+	}
+
+	slice2 := []*TwitterUser{
+		&TwitterUser{
+			Username: "muesli",
+		},
+		&TwitterUser{
+			Username: "smag",
+		},
+		&TwitterUser{
+			Username: "franz",
+		},
+	}
+
+	// Create functionality
+	list.Create(slice1, slice2)
+	assert.Equal(t, len(*list), 5, "should contain content of both slices with duplicates")
+
+	// RemoveDuplicates functionality
```

Comment:
It's good practise to put each individual test case inside a `t.Run(...)` call and describe what it's functionality it's testing there